src/main/java/net/kyori/adventure/text/minimessage/transformation/TransformationRegistry.java

```diff
@@ -143,17 +144,19 @@ private Transformation tryLoad(final Transformation transformation, final String
         throw exception;
       }
       // TODO nicer message format?
-      final Consumer<String> consumer = debugContext.miniMessage().parsingErrorMessageConsumer();
-      consumer.accept("[MiniMessage] Encountered parse exception while trying to load " + transformation.getClass().getSimpleName());
-      consumer.accept("\tmsg=" + exception.getMessage());
-      consumer.accept("\twith name=" + name + " and inners=" + inners + "");
-      consumer.accept("\tinput=" + debugContext.ogMessage());
+      final List<String> errorMessage = new ArrayList<>(Arrays.asList(
```

Comment:
Why are you wrapping the list here?

Reply:
Arrays.asList returns a fixed size list, and we sometimes add more elements after list creation